fdbserver/Knobs.cpp

```diff
@@ -363,7 +363,8 @@ void ServerKnobs::initialize(bool randomize, ClientKnobs* clientKnobs, bool isSi
 	init( START_TRANSACTION_MAX_REQUESTS_TO_START,             10000 );
 	init( START_TRANSACTION_RATE_WINDOW,                         2.0 );
 	init( START_TRANSACTION_MAX_EMPTY_QUEUE_BUDGET,             10.0 );
-	init( START_TRANSACTION_MAX_QUEUE_SIZE,                      1e6 );
+	init( START_SYSTEM_AND_DEFAULT_TRANSACTION_MAX_QUEUE_SIZE,   1e6 );
```

Comment:
Now that I see it made explicit this way, I'm wondering if it makes sense for system transactions to also be treated separately. They in general are not supposed to queue up (RK doesn't block them), so the only way you would collect them is if something in the system is causing GRVs to not be able to succeed. They are also meant to be very critical transactions, so rejecting them should only happen if there really is no alternative.